### PR TITLE
Swift: Promote the command injection query out of experimental

### PR DESCRIPTION
**Pull request by geoffw0**
Promote `swift/command-line-injection` out from experimental.  The query was added in https://github.com/github/codeql/pull/13726 and has evolved a bit while in experimental (in particular https://github.com/github/codeql/pull/13906, https://github.com/github/codeql/pull/14357 and https://github.com/github/codeql/pull/14661 - the latter is yet to be merged but should not conflict with this PR).

@maikypedia FYI

TODO:
- [x] DCA run
- [x] docs review
- [x] team review

### CODE REVIEW

**Comment by geoffw0** [comment]
DCA looks good to me.

File: swift/ql/src/change-notes/2023-11-06-command-injection.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+category: newQuery
+---
+
+* Added new query "System command built from user-controlled sources" (`swift/command-line-injection`) for Swift. This query detects system commands built from user-controlled sources without sufficient validation. The query was previously contributed to the 'experimental' directory but will now run by default for all code scanning users.
```
Comment (MathiasVP) [inline swift/ql/src/change-notes/2023-11-06-command-injection.md:5]: FYI, we've previously called out the contributor in the change note when it's been created by an external contributor. I think we should do the same here.

**Review by MathiasVP — COMMENTED** [review]
One comment, but otherwise this LGTM

**Review by subatoi — APPROVED** [review]
👍 from Docs

File: swift/ql/src/change-notes/2023-11-06-command-injection.md (hunk 1 above)
Comment (geoffw0) [reply swift/ql/src/change-notes/2023-11-06-command-injection.md]: Good idea.  Done.

**Review by MathiasVP — APPROVED** [review]
LGTM!

**Pull request merged**